### PR TITLE
Metadata fix for Arronax #304

### PR DESCRIPTION
**Pull request by anonymoussprocket**

### CODE REVIEW

File: src/main/resources/metadata/tezos.carthagenet.conf [hunk 1]
```diff
@@ -574,6 +574,10 @@
         hash {
           visible: true
           data-type: "hash"
+          reference: {
+            entity: "operations"
+            key: "operation_group_hash"
+          }
```
Comment (ivanopagano) [inline src/main/resources/metadata/tezos.carthagenet.conf]: Shouldn't be the other way round? 
The reference is from operations, referring to the containing group, via the `operation_group_hash` field.

File: src/main/resources/metadata/tezos.mainnet.conf [hunk 2]
```diff
@@ -574,6 +574,10 @@
         hash {
           visible: true
           data-type: "hash"
+          reference: {
+            entity: "operations"
+            key: "operation_group_hash"
+          }
```
Comment (ivanopagano) [inline src/main/resources/metadata/tezos.mainnet.conf]: Same as for the other file, clearly.

File: src/main/resources/metadata/tezos.carthagenet.conf (hunk 1 above)
Comment (inventorofwheels) [reply src/main/resources/metadata/tezos.carthagenet.conf]: operations is the table with the details.

Comment (anonymoussprocket) [reply src/main/resources/metadata/tezos.carthagenet.conf]: operations is the table with the details.

**Pull request merged**